CHANGELOG.md

```diff
@@ -24,6 +24,48 @@ The following API changes should be documented as such:
 If possible, provide tooling that performs the changes, e.g. a shell-script.
 -->
 
+# 3.0.2
+
+Note that 3.1.0 will be the first API stable release and interfaces in this release might still change.
+
+## New features
+
+#### Alphabet
```

Comment:
I wouldn't already add all of the modules since we might not have any changes in one of them when we do more frequent releases.

Reply:
I would give a full pre-fill, because merge conflicts will be less likely and before release we can remove not used headlines.

Reply:
This merge conflict, if it will appear, is really easy to fix so for me this reason is not strong enough to add stuff that we remove anyway. Since we have a tie I will reassign to @rrahn and he can decide.

Reply:
I agree. We should not add modules that are not likely to appear in the release. Especially, since I want to focus only one major topic for the next releases.